src/App.css

```diff
@@ -34,6 +34,43 @@
   margin-bottom: 1rem;
 }
 
+.nav-btn.back-button,
+.back-continue-buttons .back-button,
+.prev-continue-results-buttons .back-button,
+.prev-save-continue-buttons .back-button {
+  font-family: 'Open Sans';
+  color: #41528c;
+  background-color: transparent;
+  font-weight: 700;
```

Comment:
It looks like the buttons on the results page are no longer bold.

Reply:
Okay, switch font-weight to `bold`.